operators/samples/instanceRS.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: crownlabs.polito.it/v1alpha2
```

Comment:
YAML files shall be removed.

Reply:
Fixed in [e17fe5b](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/e17fe5b6dd9c3a0e1519a8924264099dfd8e21e6)